assets/css/base.css

```
@@ -0,0 +1,28 @@
+:root {
+    --white-color:#fff;
```

```suggestion
    --white-color: #fff;
```
sửa hết sau dấu : là 1 space